include/swift/AST/DefineLocalizationMacros.h

```diff
@@ -0,0 +1,41 @@
+//===- DefineLocalizationMacros.h - Localization Languages Macros -*- C++
```

Comment:
Please take a look how this  "def'inization" is done for i.e. `ConstraintLocator`. It should be sufficient to only have `LocalizationLangs.def` which would either define "default" macros or use what has already been defined.